blocks/form/file.js

```diff
@@ -0,0 +1,52 @@
+function attachFiles(wrapper, fileList, fileTemplate) {
+    for(let i = 0; i < fileList.length; i++) {
+        const file = fileList[i];
+        const input = fileTemplate.cloneNode(true);
+        input.id = input.name = 'attachment_' + Date.now() + '' + Math.floor((Math.random() * 1000));
+        const dataTransfer = new DataTransfer();
+        dataTransfer.items.add(file);
+        input.files = dataTransfer.files;
+        const div = document.createElement('div');
+        const remove = document.createElement('button');
+        remove.className = 'remove';
+        div.innerHTML = `<span>${file.name}</span>`;
```

Comment:
here we should potentially sanitize, albeit small-scale, against XSS. you could do:
```
function sanitizeHTML(input) {
    const map = {
        '&': '&amp;',
        '<': '&lt;',
        '>': '&gt;',
        '"': '&quot;',
        "'": '&#039;'
    };
    return input.replace(/[&<>"']/g, function(m) { return map[m]; });
}
```

and then use:
```
div.innerHTML = `<span>${sanitizeHTML(file.name)}</span>`;
```

Reply:
using innertext instead of innerHTML, to avoid risk of XSS.